expilcitly use python2.7 in dlpitest.py privrbactest.py ucredtest.py gami.py

http://hg.osdyson.org/solaris-desktop-spec-files/raw-file/8f7c0cd200a9/patches/gamin-02-gamin.diff

--- a/python/gamin.py.orig	2009-12-04 13:19:30.527419772 +0800
+++ b/python/gamin.py	2009-12-04 13:19:43.189517582 +0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 
 import _gamin
 import os.path
